flink-python/src/main/java/org/apache/flink/python/metric/FlinkMetricContainer.java

```diff
@@ -106,9 +106,12 @@ private void updateMetrics(String stepName) {
 
 	private boolean isUserMetric(MetricResult metricResult) {
 		MetricName metricName = metricResult.getKey().metricName();
+		String urn = ((MonitoringInfoMetricName) metricName).getUrn();
 		return (metricName instanceof MonitoringInfoMetricName) &&
-			((MonitoringInfoMetricName) metricName).getUrn()
-				.contains(MonitoringInfoConstants.Urns.USER_COUNTER);
+			(urn.contains(MonitoringInfoConstants.Urns.USER_SUM_INT64) ||
```

Comment:
Since we can ensure that metricName is `MonitoringInfoMetricName`, we can remove the judgment of metricName instanceof `MonitoringInfoMetricName`

Reply:
`MetricName` is an interface, and we can have many Implementations, and for this method `isUserMetric` we only accept the `MonitoringInfoMetricName` for now, So, I think would be OK to keep this check. 
What to you think?